pageserver/src/tenant.rs

```diff
@@ -3988,6 +4021,12 @@ pub(crate) mod harness {
         buf.freeze()
     }
 
+    pub async fn advance_time_by(duration: Duration) {
```

Comment:
This is used in unit tests to skip ahead if the behavior of the test relies on successfully doing GC but `lsn_lease_deadline` is blocking it.

Reply:
Can this function be `cfg(test)`, or is it used by a unit test in another crate?

Reply:
I added `cfg(test)`: 674969b1eb4011217c6b3399e945b8c1faf7ce88